libafl_frida/Cargo.toml

```diff
@@ -73,6 +73,7 @@ ahash = "0.8"
 paste = "1.0"
 log = "0.4.20"
 mmap-rs = "0.6.0"
+iced-x86 = { version = "1.20.0", features = ["code_asm"] }
```

Comment:
can you put optional=true

Reply:
we decided to use https://github.com/iximeow/yaxpeax-x86 
for the asan implementation

it is better if you can use this too

Reply:
it's only a decoder, I thik I can't build the instructions using it

Reply:
To build instructions, use frida `InstructionWriter` or dynasm

Reply:
>can you put optional=true
>also can make this dep as x64 only?

this should be done now, could you check if it's okay? please

> To build instructions, use frida InstructionWriter or dynasm
frida `InstructionWriter` lacks lower registers support, dynasm I seem can't use variable registers

Reply:
I think I will look into adding lower registers to `InstructionWriter`.